API.md

```diff
@@ -281,12 +281,18 @@ This gives a count of messages that have not been read for a specific user. Ther
 
 `GET /threads`
 
-This returns a list of conversations , showing the latest message in a conversation .
-This is currently implemented but not used in production. Hence should be treated with caution.
-
+This returns the latest message in each conversation(aka thread) that satisfies the passed in
+criteria.
 It can use the same filter arguments as Get Messages, and returns the latest message in each thread that satisfies
-the criteria passed in .
-[Get Message List](#get-message-list)
+the criteria passed in [Get Message List](#get-message-list)
+
+Additional parameters above those on Get Message List:  
+
+is_closed=true  if set then only returns conversations that have been marked as closed, else returns open
```

Comment:
I'd like to see these parameters in `code blocks`

Reply:
Agreed.  And same for the other newly added params